### ISSUE TITLE
Edit content via code, using Heroku one-click deployment

### ISSUE THREAD

**Issue by DominikHoffendahl** [Requarks/wiki#1597]
**Question**
I deployed the latest Wikijs on Heroku using the one-click deployment button from the [installation guide](https://docs.requarks.io/install/heroku).
I am struggling to access the repo, as I want to manage the wiki's content in a repository, rather then using the deployed page for that.
When I add the heroku remote, there is no content on that repository. How can I manage the content using the git repo?

**What I did**
`git remote add heroku git@heroku.com:project.git` ..or..
`heroku git:remote -a project`

`git pull heroku master` -> No content, no branches

**Host Info (please complete the following information):**
Basic Heroku deployment using the [installation guide](https://docs.requarks.io/install/heroku) on this very date.

**Comment by Smankusors** [comment]
wait, why do you want to edit content via code instead? Why not from Administration area?

**Comment by DominikHoffendahl** [comment]
We are all developers within the team. We want to put our markdowns in the codebase and inherit the review and acceptance functionality of the hosted repository (Github & Gitlab)

**Comment by DominikHoffendahl** [comment]
I can answer my own question. 

Within the hosted wiki instance, navigate to your administration panel. You will find it on your avatar image at the top right corner.

Navigate to the storage tab and setup GIT as your storage provider. Make sure to set it up correctly, I used SSH to manage the access.

**Use SSH**
Create a fresh SSH key pair for this. Wire up SSH correctly by adding the public key to your GitLab / GitHub account and add the private key to your GIT setup in the Wiki JS instance.

**Sync**
Sync between wikijs and your repo should be active by default. Initially, I had to import all data from DB to the local repo inside the wiki, then push those to the repo and by then the sync worked automatically. Just use the buttons at the bottom of your git settings, no coding or CLI required to achieve this.
